OptimizelySDK/Event/Internals/EventFactory.cs

```diff
@@ -0,0 +1,200 @@
+﻿/* 
```

Comment:
I don't think there is a need for making things `internal` anymore. We want to be able to expose as much of the SDK as possible. Plus the class itself it public and not protected :)

Reply:
I will remove it from `Internals`, just FYI, class should have `internal` attribute not to expose outside of assembly.